internal/data/daily.go

```diff
@@ -18,31 +19,36 @@ type DailyDownload struct {
 	Downloads int    `json:"downloads"`
 }
 
-func GetDaily(period, pkg string) (Daily, error) {
+func GetDaily(period Period, pkgs []string) ([]Daily, error) {
 	// weirdly the API returns an array of length 1
```

Comment:
this is because of that comma separated packages - keeps the same data structure if 1 package or many..... note: need to update to remove the comment now

Reply:
I did not know we could pass multiple packages